### ISSUE TITLE
tvOS iOS cloud sync breaking n64 saves

### ISSUE THREAD

**Issue by frootloop88** [libretro/RetroArch#16663]
I don't know why but at some point my safe states just can't be loaded with mupen64 core. It says error loading save state. Sometimes they get moved to the deleted folder, even if I copy them back it doesn't work anymore.

There are some things wrong with this core. Auto load safe state for example also doesn't work. And the rendering issue which forces me to kill the app is still present on iOS and tv os(already reported this)

**Comment by warmenhoven** [comment]
I have told you how to fix the rendering issue.

You can turn on logging in Settings > Logging, please turn all of the options On and all of the levels to 0,debug. The log will be created in the logs directory in the RetroArch folder.

I haven't been able to reproduce the sram or state corruption issue, so if you could create a log and attach it, that would be very helpful. Thanks.

**Comment by frootloop88** [comment]
[retroarch__2024_06_07__16_27_56.log](https://github.com/user-attachments/files/15741103/retroarch__2024_06_07__16_27_56.log)

**Comment by frootloop88** [comment]
[retroarch__2024_06_07__16_27_56.log](https://github.com/user-attachments/files/15741120/retroarch__2024_06_07__16_27_56.log)

**Comment by frootloop88** [comment]
Any idea?

**Comment by warmenhoven** [comment]
No, unfortunately I don't see anything wrong in the log, it just decides to close the content. I'll see if I can figure out a way to reproduce it.
